pts-core/external-test-dependencies/dependency-handlers/microsoft_dependency_handler.php

```diff
@@ -78,10 +78,17 @@ public static function what_provides($files_needed)
 		}
 		return $packages_needed;
 	}
-	protected static function file_download_location()
+	public static function file_download_location()
 	{
 		// TODO determine what logic may need to be applied or if to punt it as an option, etc
-		return getenv('USERPROFILE') . '\Downloads\\';
+		$download_directory = trim(shell_exec('powershell "(New-Object -ComObject Shell.Application).NameSpace(\'shell:Downloads\').Self.Path"'));
+		
+		// Fall back to user profile directory
+		if(empty($download_directory)) {
```

Comment:
Probably wise adding "|| !is_dir($download_directory)" to ensure the $download_directory is valid?